src/backend/metal/MetalBackend.h

```diff
@@ -123,17 +109,33 @@ namespace metal {
             id<MTLTexture> GetCurrentTexture();
             id<MTLTexture> GetCurrentDepthTexture();
 
+            id<MTLCommandBuffer> GetPendingCommandBuffer();
+            void SubmitPendingCommandBuffer();
+            Serial GetPendingCommandSerial();
+            Serial GetFinishedCommandSerial() const;
```

Comment:
This function is never used because `finishedCommandSerial` is a member variable which is passed to the other `Tick` functions. Are there situations where other classes will need to query the finished Serial outside of a Tick function?